src/components/Search.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import getImages from "../requests/getImages";
+import "../styles/Search.css";
+
+const Search = ({ setSearchResults }) => {
+  const [searchValue, setSearchValue] = useState("");
+  const [load, setLoad] = useState(false);
+
+  const handleSubmit = async (event) => {
+    setLoad(true);
+    event.preventDefault();
```

Comment:
Not sure if it has any performance implications, but I would always put prevent default at the start of a function so you know that the default action is being stopped before we mess with state or anything that would cause a rerender